packages/nimble-components/src/table/template.ts

```diff
@@ -59,6 +60,7 @@ export const template = html<Table>`
                             @click="${x => x.handleCollapseAllGroupRows()}"
```

Comment:
Should we also add `title` for this button so that we show a tooltip? I couldn't find documentation of a decision (the button storybook doesn't have any guidance about tooltips for icon buttons and #1090 just says we might want to do it). But my opinion is that this button especially isn't a standard icon so it could benefit from a tooltip to explain what it does.

I could go either way on the group expand/collapse button and the action menu button needing titles. I don't have justification for my sense that they don't need them other than "I can tell what they do from their icons", which is not very good justification. So maybe they need them too.

Reply:
Agreed, since this button/icon is pretty nonstandard it could probably use a title/tooltip, I'll add that.
Also agreed that the other buttons don't really need one.

Reply:
Added `title`